configuration/components/hydra/configmap.libsonnet

```diff
@@ -0,0 +1,11 @@
+{
+  apiVersion: 'v1',
+  kind: 'ConfigMap',
+  metadata: {
+    name: 'hydra-config',
+    namespace: 'hydra',
+  },
+  data: {
+    'config.yaml': 'dsn: sqlite:///var/lib/sqlite/hydra.sqlite?_fk=true\nstrategies:\n  access_token: jwt\nurls:\n  self:\n    issuer: http://hydra.hydra.svc.cluster.local:4444/',
```

Comment:
same comment here in relation to the hardcoded namespace

Reply:
@PhilipGough would it be fine if I deleted the data key completely from this file? This key is always overwritten by the manifest generation. It exists only to avoid cluttering the main hydra libsonnet with "templates".